cassandra/src/main/scala/com/stratio/crossdata/connector/cassandra/DefaultSource.scala

```diff
@@ -154,6 +157,26 @@ class DefaultSource extends CassandraConnectorDS with TableInventory with Functi
 
   }
 
+  override def createExternalTable(context: SQLContext, tableName: String, schema: StructType, options: Map[String, String]): Boolean = {
+
+    val keyspace: String = {
+      require(options.contains(CassandraDataSourceKeyspaceNameProperty),
+        s"$CassandraDataSourceKeyspaceNameProperty required when use CREATE EXTERNAL TABLE command")
+      options.get(CassandraDataSourceKeyspaceNameProperty).get
+    }
+
+    buildCassandraConnector(context, options).withSessionDo { s =>
```

Comment:
Concerning the return value, I'd make it depend on whether the creation was successful or not, something like:

```
override def createExternalTable(context: SQLContext, tableName: String, schema: StructType, options: Map[String, String]): Boolean = {

  val keyspace: String = {
    require(options.contains(CassandraDataSourceKeyspaceNameProperty),
      s"$CassandraDataSourceKeyspaceNameProperty required when use CREATE EXTERNAL TABLE command")
    options.get(CassandraDataSourceKeyspaceNameProperty).get
  }

  Try {
   buildCassandraConnector(context, options).withSessionDo { s =>

    if (s.getCluster.getMetadata.getKeyspace(keyspace)== null){
      val createKeyspace = new CreateKeyspaceStatement(options)
      s.execute(createKeyspace.toString())
    }

    val stm = new CreateTableStatement(tableName, schema, options)
    s.execute(stm.toString())
   }
   true
  } recover { case e: Exception =>
    sys.error(e.getMessage)
    false
  } get
  
}
```